js/repl/PluginConfig.js

```diff
@@ -79,7 +79,7 @@ const pluginConfigs: Array<PluginConfig> = [
     baseUrl: "https://unpkg.com",
     label: "Prettify",
     package: "prettier",
-    version: "1.13.0",
+    version: "2",
```

Comment:
This will ensure we are always using latest prettier 2. We can always pin it if any non-major release of prettier breaks our REPL.

Reply:
The babel parser filename has been renamed, related #2274.